swagger/swagger.yaml

```diff
@@ -1905,6 +1901,13 @@ components:
         - SEARCH
         - SITREP
         - TAG_CHARACTER
+    BloodOxygenEnum:
+      type: string
+      description: The percentage of oxygen in someone's blood, as measured by a pulse oximeter
+      enum:
+        - NORMAL
+        - LOW
+        - NONE
```

Comment:
I know the description mentioned NONE for severe hypoxia when the blood oxygen level may be undetectable, but is it worth possibly adding a seperate enum for severe when the blood oxygen is below 85% and potentially unreliable and then one enum for none/invalid for when reading is unavailable and outside the measurable range?

Reply:
We're changing this to be in sync with the sim, so we'd need to add another category there, too.  Not that that's impossible, but it would need to be coordinated with the sim side-- and also probably with our medical SME from OSU (Nick Kman).

Reply:
I believe we want it to match the simulator which only has those three values

Reply:
Got it, that makes sense.